internal/util/csiconfig_test.go

```diff
@@ -478,3 +482,95 @@ func TestGetReadAffinityOptions(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCephFSMountOptions(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name                 string
+		clusterID            string
+		wantKernelMntOptions string
+		wantFuseMntOptions   string
+	}{
+		{
+			name:                 "cluster-1 with non-empty mount options",
+			clusterID:            "cluster-1",
+			wantKernelMntOptions: "crc",
+			wantFuseMntOptions:   "ro",
+		},
+		{
+			name:                 "cluster-2 with empty mount options",
+			clusterID:            "cluster-2",
+			wantKernelMntOptions: "",
+			wantFuseMntOptions:   "",
+		},
+		{
+			name:                 "cluster-3 with no mount options",
+			clusterID:            "cluster-3",
+			wantKernelMntOptions: "",
+			wantFuseMntOptions:   "",
+		},
+	}
+
+	csiConfig := []ClusterInfo{
+		{
+			ClusterID: "cluster-1",
+			CephFS: struct {
+				NetNamespaceFilePath string `json:"netNamespaceFilePath"`
+				SubvolumeGroup       string `json:"subvolumeGroup"`
+				KernelMountOptions   string `json:"kernelMountOptions"`
+				FuseMountOptions     string `json:"fuseMountOptions"`
+			}{
+				KernelMountOptions: "crc",
```

Comment:
Mention the input along the output in the same struct or create a sub struct for it.

Reply:
We require the separate definition of 'csiConfig' input since it will be written to a file. It will be complex to combine both input and output together.